app/models/root-store/root-store.ts

```diff
@@ -91,11 +93,12 @@ export const RootStoreModel = types
     }),
   }))
   .actions(self => ({
-    handleUnauthenticatedError() {
+    handleUnauthenticatedError(type: string) {
+      logAnalyticsEvent("UnauthenticatedError", { type })
```

Comment:
I think can keep passing the error object in the interfaces, and maybe event pass error.messge/toString()  as analytic payload